src/gui/Pages/Chat/RoomPane/MemberView/MemberDelegate.qml

```diff
@@ -24,25 +25,64 @@ HTile {
             shiftMembershipIconPosition: ! roomPane.collapsed
             invited: model.invited
             compact: member.compact
+            presence: model.presence
         }
 
         HColumnLayout {
-            TitleLabel {
-                text: model.display_name || model.id
-                color:
-                    member.hovered ?
-                    utils.nameColor(
-                        model.display_name || model.id.substring(1)
-                    ) :
-                    theme.chat.roomPane.listView.member.name
-
-                Behavior on color { HColorAnimation {} }
+            HRowLayout {
+                TitleLabel {
+                    text: model.display_name || model.id
+                    color:
+                        member.hovered ?
+                        utils.nameColor(
+                            model.display_name || model.id.substring(1)
+                        ) :
+                        theme.chat.roomPane.listView.member.name
+
+                    Behavior on color { HColorAnimation {} }
+                }
+
+                TitleRightInfoLabel {
+                    tile: member
+                    visible: presenceTimer.running
+
+                    Timer {
+                        id: presenceTimer
+                        running:
+                            ! model.currently_active &&
+                            model.last_active_at > new Date(1)
+                        repeat: true
+                        interval:
+                            new Date() - model.last_active_at < 60000 ?
+                            10000 :
+                            60000
+                        triggeredOnStart: true
+                        onTriggered: parent.text = Qt.binding(() =>
+                            utils.formatRelativeTime(
+                                new Date() - model.last_active_at
+                            )
```

Comment:
You might want to move the Timer outside of the label to avoid all the line breaking due to indentation